packages/ra-core/src/controller/record/RecordRepresentation.spec.tsx

```diff
@@ -10,7 +10,7 @@ import {
 describe('RecordRepresentation', () => {
     it('should render the record id when not provided on its parent <Resource>', async () => {
         render(<NoRecordRepresentation />);
-        await screen.findByText('#1');
+        await screen.findByText("The Hitchhiker's Guide to the Galaxy");
```

Comment:
<!-- Polish comments are like a suggestion, where there is nothing necessarily wrong with the relevant content, there's just some ways to immediately improve the quality.  -->
**polish:** ‏The test description should be updated accordingly